Cargo.toml

```diff
@@ -22,6 +22,6 @@ kittycad = { version = "0.2.44", features = ["requests"] }
 kittycad-execution-plan = { path = "execution-plan" }
 kittycad-execution-plan-macros = { path = "execution-plan-macros", version = "0.1.8" }
 kittycad-execution-plan-traits = { path = "execution-plan-traits", version = "0.1.12" }
-kittycad-modeling-cmds = { path = "modeling-cmds", version = "0.1.24" }
+kittycad-modeling-cmds = { path = "modeling-cmds", version = "0.2.0" }
```

Comment:
We're not going to release an 0.2.0 because it'd break the current engine/format setup -- just use patch bumps, it's fine :)

Reply:
I was getting semver complaints when not updating the minor version / only updating the patch version

Reply:
Reverted to patch bump